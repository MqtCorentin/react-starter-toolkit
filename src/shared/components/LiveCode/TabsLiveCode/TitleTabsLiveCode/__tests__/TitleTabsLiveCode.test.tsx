import { render } from '@testing-library/react';
import TitleTabsLiveCode from '../TitleTabsLiveCode';

const propsMock = {
  icon: 'ok',
  title: 'title',
};
describe('<TabsLiveCode />', () => {
  it('Render <TabsLiveCode /> with props', () => {
    const { asFragment } = render(<TitleTabsLiveCode props={propsMock} />);

    expect(asFragment()).toMatchSnapshot();
  });
});
