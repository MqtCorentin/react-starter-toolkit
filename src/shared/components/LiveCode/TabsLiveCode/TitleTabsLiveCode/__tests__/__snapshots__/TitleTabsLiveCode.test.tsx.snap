// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TabsLiveCode /> Render <TabsLiveCode /> with props 1`] = `
<DocumentFragment>
  <i
    class="glyphicon glyphicon-undefined"
  />
   
</DocumentFragment>
`;
